CuratedContent/bssw-tutorials.md

```diff
@@ -0,0 +1,26 @@
+### Better Scientific Software (BSSw) Tutorial
+<!-- deck text start --> 
+
+Keeping skills up to date and focused helps keep work at its best. The BSSw Tutorial page presents a vertical sampler of upcoming and past tutorials on improving scientific software.
+
+#### Contributed by [Deborah Stevens](https://github.com/haikudeb)
+#### Publication date: May 08, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title | [The Better Scientific Software (BSSw) Tutorial](https://bssw-tutorial.github.io)
+Authors | [Deborah Stevens](https://github.com/haikudeb)
```

Comment:
Here, we reference the author(s) of there resource we are referring to and not the author of this CC. I think listing just David here is appropriate because he has spearheaded so much of the activity on the tutorial(s) and the website hosting them.

```suggestion
Authors | [David Bernholdt](https://github.com/bernhold) et al. 2016-
```

Reply:
I see @bernhold emailed a different way he'd like this and @rinkug wants a different table too. I will *resolve* this conversation and add a new one with both of their requests combined.